drakrun/drakrun/draksetup.py

```diff
@@ -245,6 +246,79 @@ def send_usage_report(report):
         logging.exception("Failed to send usage report. This is not a serious problem.")
 
 
+def create_rekall_profiles(install_info, pdb_guid):
```

Comment:
I would extract part of the function that performs the PDB download and profile generation into a new one that takes a path to the mounted directory as an argument

Reply:
Would make sense if you wanted to provide your own DLLs, but that's not the case (since you can't provide additional rekall profiles to drakvuf, since if you wanted to do that you'd have to use apimon anyways).